vm/src/vm.rs

```diff
@@ -166,7 +168,10 @@ impl VirtualMachine {
         let import_func = RefCell::new(ctx.none());
         let profile_func = RefCell::new(ctx.none());
         let trace_func = RefCell::new(ctx.none());
+
         let vm = VirtualMachine {
+            #[cfg(not(target_arch = "wasm32"))]
```

Comment:
We can have this signal handlers in wasm as well right? They do not hurt us there, so this `#[cfg]` could be removed.

Reply:
It uses the `NSIG` constant from `signal.rs`; should I move that into `vm.rs`?

Reply:
Yes, this is probably a good idea to keep the `NSIG` constant where the array is defined / created.